Experiments

kariekirkpatrick opened the pull request:
Adding my advice and practicing GH

redrambles commented:
Hey @kariekirkpatrick !

This is great - there are just a few little changes that we need you to make before we are good to merge this in.

1) You are linking to css/style.kariek-advice.css from your advice page - but you have also added styling to the main style.css file. You'll only need one of these! If you decide to keep your custom stylesheet, please remove your styles from the style.css file. If you wish to use the style.css file, please delete the css/style.kariek-advice.css file and make sure to link to style.css from your advice page.

2) You have a second 'skillcrush-104' directory inside your first skillcrush-104 directory, which is being added as a submodule. You can see it listed amongst your changes [here](https://github.com/skillcrush/skillcrush-104/pull/1827/files). Please delete that second 'skillcrush-104' folder.

Once this is done - add `git add -A` and commit `git commit -m "your message here"` your changes and then push up your skillcrush-104 branch once more: `git push origin skillcrush-104`. This will automatically update your pull request, and we should be all set to merge in your code.

Let us know if you have any questions!
Cheers,
Ann

kariekirkpatrick commented:
Ann,

I deleted my custom stylesheet and linked to the css/style.css file. Then I deleted that subfolder (how did that get there??). I added and committed my changes in terminal, then tried 'git push origin skillcrush-104' and got the message:
 
error: src refspec skillcrush-104 does not match any.
error: failed to push some refs to 'git@github.com:kariekirkpatrick/skillcrush-104.git'

So I tried 'git push origin experiments', and that seemed to be fine but I have no idea if it updated my pull request or not. How can I know? 
Also, do I need to do anything about the above message that says there are conflicts that need to be solved on my branch? It looks like it has to do with people that added changes after me.

Thanks again,
Karie

redrambles commented:
Hey @kariekirkpatrick ,

You did a great! 'git push origin experiments' is exactly what needed to be done, once you made those corrections. Everything looks good, your code is all merged in! Yay!

Cheers,
Ann :)
